tests/test_sentry.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+
+def test_generate_logcheck_error(conf, env):
+    response = requests.get(conf.get(env, 'server_url') + "/v1/err/crit")
```

Comment:
I'm already working on this test here:
https://github.com/rpappalax/autopush-integration-tests/issues/15
so you can delete this module

Reply:
I still need this test to be around until your version is completed

Reply:
This test will trigger sentry, but won't 'check' it, but perhaps you know that already.  As long as you still check the Sentry page manually then yea, we can leave it in.